language-server/modules/langserver-commons/src/main/java/org/ballerinalang/langserver/commons/codeaction/spi/DiagBasedPositionDetails.java

```diff
@@ -51,4 +53,14 @@
      * @return Value of the property at given index
      */
     <T> Optional<T> diagnosticProperty(int propertyIndex);
+
+    /**
+     * Returns property index given the code.
+     * 
+     * @param diagnosticCode Diagnostic code.
+     * @param propertyName Name of the property.
+     * @return
```

Comment:
Shall we complete the @return line? We can may be change the line 58 to `Get property index given the code.`

Reply:
Done